packages/connect/src/api/blockchainEstimateFee.ts

```diff
@@ -44,6 +44,7 @@ export default class BlockchainEstimateFee extends AbstractMethod<'blockchainEst
                     { name: 'to', type: 'string' },
                     { name: 'txsize', type: 'number' },
                     { name: 'isCreatingAccount', type: 'boolean' },
+                    { name: 'creatingTokenAccountType', type: 'string' },
```

Comment:
I think we might want to validate that the string is indeed valid i.e. of `TokenProgramName`. I'd add such validation and if it's not needed, Trezor guys will let us know. Or don't add it and ask about it in their PR 🙏

Reply:
I don't see validation of similar "string union" fields anywhere else, I'll ask in the PR.